lib/Jijo/Definition.hs

```diff
@@ -374,6 +376,14 @@ jNullable jDef = jDefinition validateNullable encodeNullable
       JTypeNotOneOf (Set.insert JTyNull jtys)
     adjustErr e = e
 
+jList :: forall e a. JDefinition e JSON.Value a -> JDefinition e JSON.Value [a]
+jList def = jDefinition from to . jArray
+  where
+    from :: JSON.Array -> JValidation e [a]
+    from xs = sequenceA (Vect.toList (Vect.map (jValidate def) xs))
```

Comment:
This isn't quite as easy as that. The validation report must be extended with a corresponding `PathSegment` using `scopeJValidationReport`. For example, if the user validates `[1,2,"hi"]` against `jList jNumber`, the path to the error must be `[JPSIndex 2]` (the index of the string), not `[]`.